<div class="header">
  <h1 class="page-title">
    <a class="icon circle-icon glyphicon glyphicon-chevron-left trigger-message-close"></a>
    FASTQ格式详解
  </h1>
  <div class="details">
    <div class="left">
      7月 27, 2015
      <span> | </span> <a href="/category/li-lun-ji-chu.html">理论基础</a>
    </div>
    <div class="right">
      <a class="fa fa-tag" href="/tag/fastq.html">fastq</a>
    </div>
  </div>
</div>
<div id="message-nano-wrapper" class="nano">
  <div class="nano-content">
    <ul class="message-container">
        <div class="message">
          <p>FASTQ是基于文本的，保存生物序列（通常是核酸序列）和其测序质量信息的标准格式。
其序列以及质量信息都是使用一个ASCII字符标示，最初由Sanger开发，目的是将FASTA序列与质量数据放到一起，目前已经成为高通量测序结果的事实标准。</p>
<h1>格式说明</h1>
<p>FASTQ文件中每个序列通常有四行：
1.序列标识以及相关的描述信息，以‘@’开头；
2.第二行是序列
3.第三行以‘+’开头，后面是序列标示符、描述信息，或者什么也不加
4.第四行，是质量信息，和第二行的序列相对应，每一个序列都有一个质量评分，根据评分体系的不同，每个字符的含义表示的数字也不相同。</p>
<p>例如：</p>
<div class="highlight"><pre><span class="code-line"><span></span>  @SEQ_ID</span>
<span class="code-line">  GATTTGGGGTTCAAAGCAGTATCGATCAAATAGTAAATCCATTTGTTCAACTCACAGTTT</span>
<span class="code-line">  <span class="se">\+</span>  </span>
<span class="code-line">  !<span class="s1">&#39;&#39;</span>*<span class="o">((((</span>***+<span class="o">))</span>%%%++<span class="o">)(</span>%%%%<span class="o">)</span>.1***-+*<span class="s1">&#39;&#39;</span><span class="o">))</span>**55CCF&gt;&gt;&gt;&gt;&gt;&gt;CCCCCCC65</span>
</pre></div>


<h2>Illumina sequence identifiers</h2>
<blockquote>
<p><strong>@HWUSI-EAS100R:6:73:941:1973#0/1</strong></p>
</blockquote>
<table>
<thead>
<tr>
<th align="center">HWUSI-EAS100R</th>
<th>the unique instrument name</th>
</tr>
</thead>
<tbody>
<tr>
<td align="center">6</td>
<td>flowcell lane</td>
</tr>
<tr>
<td align="center">73</td>
<td>tile number within the flowcell lane</td>
</tr>
<tr>
<td align="center">941</td>
<td>‘x’-coordinate of the cluster within the tile</td>
</tr>
<tr>
<td align="center">1973</td>
<td>‘y’-coordinate of the cluster within the tile</td>
</tr>
<tr>
<td align="center">#0</td>
<td>index number for a multiplexed sample (0 for no indexing)</td>
</tr>
<tr>
<td align="center">/1</td>
<td>the member of a pair, /1 or /2 _(paired-end or mate-pair reads only)</td>
</tr>
</tbody>
</table>
<blockquote>
<p><strong>@EAS139:136:FC706VJ:2:2104:15343:197393 1:Y:18:ATCACG</strong></p>
</blockquote>
<table>
<thead>
<tr>
<th align="center">EAS139</th>
<th>the unique instrument name</th>
</tr>
</thead>
<tbody>
<tr>
<td align="center">136</td>
<td>the run id</td>
</tr>
<tr>
<td align="center">FC706VJ</td>
<td>the flowcell id</td>
</tr>
<tr>
<td align="center">2</td>
<td>flowcell lane</td>
</tr>
<tr>
<td align="center">2104</td>
<td>tile number within the flowcell lane</td>
</tr>
<tr>
<td align="center">15343</td>
<td>‘x’-coordinate of the cluster within the tile</td>
</tr>
<tr>
<td align="center">197393</td>
<td>‘y’-coordinate of the cluster within the tile</td>
</tr>
<tr>
<td align="center">1</td>
<td>the member of a pair, 1 or 2 <em>(paired-end or mate-pair reads only)</em></td>
</tr>
<tr>
<td align="center">Y</td>
<td>Y if the read fails filter (read is bad), N otherwise</td>
</tr>
<tr>
<td align="center">18</td>
<td>0 when none of the control bits are on, otherwise it is an even number</td>
</tr>
<tr>
<td align="center">ATCACG</td>
<td>index sequence</td>
</tr>
</tbody>
</table>
<h3>NCBI Sequence Read Archive</h3>
<div class="highlight"><pre><span class="code-line"><span></span>  @SRR001666.1 071112_SLXA-EAS1_s_7:5:1:817:345 <span class="nv">length</span><span class="o">=</span><span class="m">36</span></span>
<span class="code-line">  GGGTGATGGCCGCTGCCGATGGCGTCAAATCCCACC  </span>
<span class="code-line">  +SRR001666.1 071112_SLXA-EAS1_s_7:5:1:817:345 <span class="nv">length</span><span class="o">=</span><span class="m">36</span>  </span>
<span class="code-line">  IIIIIIIIIIIIIIIIIIIIIIIIIIIIII9IG9IC</span>
</pre></div>


<h2>关于质量编码格式</h2>
<p>质量评分指的是一个碱基的错误概率的对数值。其最初在Phred拼接软件中定义与使用，其后在许多软件中得到使用。其质量得分与错误概率的对应关系见下表：</p>
<ul>
<li>Phred quality scores are logarithmically linked to error probabilities Phred</li>
</ul>
<table>
<thead>
<tr>
<th>Quality Score</th>
<th>Probability of incorrect base call</th>
<th>Base call accuracy</th>
</tr>
</thead>
<tbody>
<tr>
<td>10</td>
<td>1 in 10</td>
<td>90 %</td>
</tr>
<tr>
<td>20</td>
<td>1 in 100</td>
<td>99 %</td>
</tr>
<tr>
<td>30</td>
<td>1 in 1000</td>
<td>99.9 %</td>
</tr>
<tr>
<td>40</td>
<td>1 in 10000</td>
<td>99.99 %</td>
</tr>
<tr>
<td>50</td>
<td>1 in 100000</td>
<td>99.999 %</td>
</tr>
<tr>
<td>&nbsp;</td>
<td></td>
<td></td>
</tr>
</tbody>
</table>
<p>$$Q=-10log_{10}P$$</p>
<blockquote>
<p>Q: Phred quality scores
P: base-calling error probabilities</p>
</blockquote>
<p>除了Phred质量得分换算标准，还有就是Solexa标准：
$$Q_{solexa-prior to v.1.3}=-10log_{10}\frac{P}{1-P}$$</p>
<p>两种换算标准的比较：
<img alt="center" src="../images/compare.png" />
Relationship between Q and p using the Sanger (red) and Solexa (black)
equations (described above). The vertical dotted line indicates p = 0.05, or
equivalently, Q ≈ 13.</p>
<ul>
<li>
<p>对于每个碱基的质量编码标示，不同的软件采用不同的方案，目前有5种方案：</p>
</li>
<li>
<p>Sanger，Phred quality score，值的范围从0到92，对应的ASCII码从33到126，但是对于测序数据（raw read data）质量得分通常小于60，序列拼接或者mapping可能用到更大的分数。</p>
</li>
<li>
<p>Solexa/Illumina 1.0, Solexa/Illumina quality score，值的范围从-5到63，对应的ASCII码从59到126，对于测序数据，得分一般在-5到40之间；</p>
</li>
<li>Illumina 1.3+，<a href="http://en.wikipedia.org/wiki/Phred_quality_score">Phred quality score</a>，值的范围从0到62对应的ASCII码从64到126，低于测序数据，得分在0到40之间；</li>
<li>Illumina 1.5+，Phred quality score，但是0到2作为另外的标示，<a href="http://solexaqa.sourceforge.net/questions.htm#illumina">详见</a></li>
<li>Illumina 1.8+</li>
</ul>
<p>下面是更为直观的表示：</p>
<div class="highlight"><pre><span class="code-line"><span></span>SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS.....................................................</span>
<span class="code-line">..........................XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX......................</span>
<span class="code-line">...............................IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII......................</span>
<span class="code-line">.................................**J**JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ......................</span>
<span class="code-line">LLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLL....................................................</span>
<span class="code-line">!<span class="s2">&quot;#</span>$<span class="s2">%&amp;&#39;()*+,-./0123456789:;&lt;=&gt;?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\]^_`abcdefghijklmnopqrstuvwxyz{|}~</span></span>
<span class="code-line"><span class="s2">|                         |    |        |                              |                     |</span></span>
<span class="code-line"><span class="s2">33                        59   64       73                            104                   126</span></span>
<span class="code-line"></span>
<span class="code-line"><span class="s2">S - Sanger        Phred+33,  raw reads typically (0, 40)</span></span>
<span class="code-line"><span class="s2">X - Solexa        Solexa+64, raw reads typically (-5, 40)</span></span>
<span class="code-line"><span class="s2">I - Illumina 1.3+ Phred+64,  raw reads typically (0, 40)</span></span>
<span class="code-line"><span class="s2">J - Illumina 1.5+ Phred+64,  raw reads typically (3, 40)</span></span>
<span class="code-line"><span class="s2">  with 0=unused, 1=unused, 2=Read Segment Quality Control Indicator (bold)</span></span>
<span class="code-line"><span class="s2">  (Note: See discussion above).</span></span>
<span class="code-line"><span class="s2">L - Illumina 1.8+ Phred+33,  raw reads typically (0, 41)</span></span>
</pre></div>
        </div>
        <!-- <div class="tool-box">
          <a href="#" class="circle-icon small glyphicon glyphicon-share-alt"></a>
          <a href="#" class="circle-icon small red-hover glyphicon glyphicon-remove"></a>
          <a href="#" class="circle-icon small red-hover glyphicon glyphicon-flag"></a>
        </div> -->
    </ul>
  </div>
</div>

<script>
  $('.nano').overlayScrollbars({
    scrollbars: {
      autoHide: "leave",
    }
  });
  $('.trigger-message-close').on('click', function () {
    $('body').removeClass('show-message');
    $('#main .message-list li').removeClass('active');
    messageIsOpen = false;
    $('body').removeClass('show-main-overlay');
  });
  $('#message .header .left a').on('click', function (e) {
    e.preventDefault();
    helper.setQuery().clearRefinements().addDisjunctiveFacetRefinement("category", e.target.text).search();
    
  });
  $('#message .header .right a').on('click', function (e) {
    e.preventDefault();
    helper.setQuery().clearRefinements().addFacetRefinement("tags", e.target.text).search();
  });
</script>