<div class="header">
  <h1 class="page-title">
    <a class="icon circle-icon glyphicon glyphicon-chevron-left trigger-message-close"></a>
    突变分类及定义功能
  </h1>
  <div class="details">
    <div class="left">
      11月 05, 2015
      <span> | </span> <a href="/category/sheng-wu-xin-xi.html">生物信息</a>
    </div>
    <div class="right">
      <a class="fa fa-tag" href="/tag/mutation.html">mutation</a>
    </div>
  </div>
</div>
<div id="message-nano-wrapper" class="nano">
  <div class="nano-content">
    <ul class="message-container">
        <div class="message">
          <h1>概念</h1>
<p><strong>突变与变异是两个截然不同的概念</strong>。<strong>变异</strong>一般是指物种在漫长的自然选择压力下，由于“适者生存”的原则而使自身遗传基因发生的某些有益于自身完善于生存环境而生存、发展的变化。</p>
<p><strong>突变</strong>则是指物种遗传基因在某些物理、化学、生物因素作用下，短期内发生的某些基因序列的变化。就人体而言，突变一般是对人体十分有害的。</p>
<p><strong>突变可以分为3大类型</strong>：点突变（point mutation）、染色体突变(chromosomal mutation)和基因组突变(genomic mutation)。 </p>
<ol>
<li>其中以点突变最为常见和重要，包括转换(transition)、颠换（transversion）、插入(insertion)和缺失(deletion)几种类型，前两种属于碱基置换(basesubstitution)，后两种属于移码突变(frame shift mutation) 。</li>
</ol>
<p>substitution（替换）和delection/addition(添加/缺失），估计就是碱基置换、移码、缺失和插入，只是翻译不同，这种分类有点笼统了，适合用于定义{基因突变是指DNA分子中发生碱基对的替换、增添和缺失，而引起的基因结构的改变，它包括单个碱基改变所引起的点突变（point mutation），或多个碱基的缺失、重复和插入。}   </p>
<p>不过更笼统的是按照遗传信息的改变方式，分为错义、无义两类。   </p>
<p>我看的分类复杂点。我看过的是类似是这个分类，你去看下网站。好象是上海一个大学的。   </p>
<p>基因突变的类型 <br>
<a href="http://genetics.sjtu.edu.cn/genetics/21.01.htm">http://genetics.sjtu.edu.cn/genetics/21.01.htm</a>   </p>
<p>突变的原因   </p>
<p><a href="http://genetics.sjtu.edu.cn/genetics/21.02.htm">http://genetics.sjtu.edu.cn/genetics/21.02.htm</a>   </p>
<p>根据基因结构的改变方式不同，可将基因突变分为四种类型： <br>
（1）点突变 由某位点一对减基改变造成的。其包括两种形式：转换和颠换。点突变的不同效应为：①同义突变；②错义突变；③无义突变；④终止密码突变。 <br>
（2）移码突变 某位点增添或减少1～2对碱基造成的。 <br>
（3）缺失突变 基因内部缺失某个DNA小段造成的。 <br>
（4）插入突变 基因内部增添一小段外源DNA造成的。   </p>
<p>基因突变是随机发生的。它可以发生在生物个体发育的任何时期和生物体的任何细胞。一般来说，在生物个体发育的过程中，基因突变发生的时期越迟，生物体表现突变的部分就越少。例如，植物的叶芽如果在发育的早期发生基因突变，那么由这个叶芽长成的枝条，上面着生的叶、花和果实都有可能与其他枝条不同。如果基因突变发生在花芽分化时，那么，将来可能只在一朵花或一个花序上表现出变异。 <br>
基因突变可以发生在体细胞中，也可以发生在生殖细胞中。发生在生殖细胞中的突变，可以通过受精作用直接传递给后代。发生在体细胞中的突变，一般是不能传递给后代的。 <br>
基因突变在生物界中是普遍存在的。无论是低等生物，还是高等的动植物以及人，都可能发生基因突变。基因突变在自然界的物种中广泛存在。例如，棉花的短果枝、水稻的矮杆、糯性，果蝇的白眼、残翅，家鸽羽毛的灰红色，以及人的色肓、糖尿病、白化病等遗传病，都是突变性状。自然条件下发生的基因突变叫做自然突变，人为条件下诱发产生的基因突变叫做诱发突变。   </p>
<p>绝大多数的人类遗传病，就是由基因突变造成的，这些病对人类健康构成了严重威胁。又如，植物中常见的白化苗，也是基因突变形成的。这种苗由于缺乏叶绿素，不能进行光合作用制造有机物，最终导致死亡。但是，也有少数基因突变是有利的。例如，植物的抗病性突变、耐旱性突变、微生物的抗药性突变等，都是有利于生物生存的。</p>
<p>nonsense mutation 无义突变：使原本可制造蛋白质的密码变成停止密码</p>
<p>missense mutation 错义突变：使密码所对应氨基酸改变</p>
<p>silent mutation 无表型突变：密码改变，但对应的氨基酸不变</p>
<p>INV， inversion 倒位 、转置</p>
<p>Interchromosomal translocation, CTX 染色体间易位  </p>
<p>intra-chromosome translocation , ITX 染色体内易位</p>
<table>
<thead>
<tr>
<th align="center">SO term</th>
<th align="center">description</th>
</tr>
</thead>
<tbody>
<tr>
<td align="center"><a href="http://sequenceontology.org/browser/current_release/term/SO:0001628">intergenic_variant</a></td>
<td align="center">A sequence variant located in the intergenic region, between genes.</td>
</tr>
<tr>
<td align="center"><a href="http://sequenceontology.org/browser/current_release/term/SO:0001631">upstream_gene_variant</a></td>
<td align="center">A sequence variant located 5' of a gene. (VAI searches within 5,000 bases.)</td>
</tr>
<tr>
<td align="center"><a href="http://sequenceontology.org/browser/current_release/term/SO:0001632">downstream_gene_variant</a></td>
<td align="center">A sequence variant located 3' of a gene. (VAI searches within 5,000 bases.)</td>
</tr>
<tr>
<td align="center"><a href="http://sequenceontology.org/browser/current_release/term/SO:0001623">5_prime_UTR_variant</a></td>
<td align="center">A variant located in the 5' untranslated region (UTR) of a gene.</td>
</tr>
<tr>
<td align="center"><a href="http://sequenceontology.org/browser/current_release/term/SO:0001624">3_prime_UTR_variant</a></td>
<td align="center">A variant located in the 3' untranslated region (UTR) of a gene.</td>
</tr>
<tr>
<td align="center"><a href="http://sequenceontology.org/browser/current_release/term/SO:0001819">synonymous_variant</a></td>
<td align="center">A sequence variant where there is no resulting change to the encoded amino acid.</td>
</tr>
<tr>
<td align="center"><a href="http://sequenceontology.org/browser/current_release/term/SO:0001583">missense_variant</a></td>
<td align="center">A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.</td>
</tr>
<tr>
<td align="center"><a href="http://sequenceontology.org/browser/current_release/term/SO:0001821">inframe_insertion</a></td>
<td align="center">An inframe non synonymous variant that inserts bases into in the coding sequence.</td>
</tr>
<tr>
<td align="center"><a href="http://sequenceontology.org/browser/current_release/term/SO:0001822">inframe_deletion</a></td>
<td align="center">An inframe non synonymous variant that deletes bases from the coding sequence.</td>
</tr>
<tr>
<td align="center"><a href="http://sequenceontology.org/browser/current_release/term/SO:0001589">frameshift_variant</a></td>
<td align="center">A sequence variant which causes a disruption of the translational reading frame, because the number of nucleotides inserted or deleted is not a multiple of three.</td>
</tr>
<tr>
<td align="center"><a href="http://sequenceontology.org/browser/current_release/term/SO:0001582">initiator_codon_variant</a></td>
<td align="center">A codon variant that changes at least one base of the first codon of a transcript.</td>
</tr>
<tr>
<td align="center"><a href="http://sequenceontology.org/browser/current_release/term/SO:0001626">incomplete_terminal_codon_variant</a></td>
<td align="center">A sequence variant where at least one base of the final codon of an incompletely annotated transcript is changed.</td>
</tr>
<tr>
<td align="center"><a href="http://sequenceontology.org/browser/current_release/term/SO:0001578">stop_lost</a></td>
<td align="center">A sequence variant where at least one base of the terminator codon (stop) is changed, resulting in an elongated transcript.</td>
</tr>
<tr>
<td align="center"><a href="http://sequenceontology.org/browser/current_release/term/SO:0001567">stop_retained_variant</a></td>
<td align="center">A sequence variant where at least one base in the terminator codon is changed, but the terminator remains.</td>
</tr>
<tr>
<td align="center"><a href="http://sequenceontology.org/browser/current_release/term/SO:0001572">exon_loss</a></td>
<td align="center">A sequence variant whereby an exon is lost from the transcript. (VAI assigns this term when an entire exon is deleted.)</td>
</tr>
<tr>
<td align="center"><a href="http://sequenceontology.org/browser/current_release/term/SO:0001587">stop_gained</a></td>
<td align="center">A sequence variant whereby at least one base of a codon is changed, resulting in a premature stop codon, leading to a shortened transcript.</td>
</tr>
<tr>
<td align="center"><a href="http://sequenceontology.org/browser/current_release/term/SO:0001621">NMD_transcript_variant</a></td>
<td align="center">A variant in a transcript that is already the target of nonsense-mediated decay (NMD), i.e. stop codon is not in last exon nor within 50 bases of the end of the second-to-last exon.</td>
</tr>
<tr>
<td align="center"><a href="http://sequenceontology.org/browser/current_release/term/SO:0001627">intron_variant</a></td>
<td align="center">A transcript variant occurring within an intron.</td>
</tr>
<tr>
<td align="center"><a href="http://sequenceontology.org/browser/current_release/term/SO:0001575">splice_donor_variant</a></td>
<td align="center">A splice variant that changes the 2-base region at the 5' end of an intron.</td>
</tr>
<tr>
<td align="center"><a href="http://sequenceontology.org/browser/current_release/term/SO:0001574">splice_acceptor_variant</a></td>
<td align="center">A splice variant that changes the 2 base region at the 3' end of an intron.</td>
</tr>
<tr>
<td align="center"><a href="http://sequenceontology.org/browser/current_release/term/SO:0001630">splice_region_variant</a></td>
<td align="center">A sequence variant in which a change has occurred within the region of the splice site, either within 1-3 bases of the exon or 3-8 bases of the intron.</td>
</tr>
<tr>
<td align="center"><a href="http://sequenceontology.org/browser/current_release/term/SO:0001577">complex_transcript_variant</a></td>
<td align="center">A transcript variant with a complex insertion or deletion (indel) that spans an exon/intron border or a coding sequence/UTR border.</td>
</tr>
<tr>
<td align="center"><a href="http://sequenceontology.org/browser/current_release/term/SO:0001792">non_coding_exon_variant</a></td>
<td align="center">A sequence variant that changes exon sequence of a non-coding gene.</td>
</tr>
</tbody>
</table>
<h3>Functional role</h3>
<p>By default, all variants are included in the output regardless of predicted
functional effect. If you would like to keep only variants that have a
particular type of effect, you can uncheck the checkboxes of other effect
types. The detailed functional effect predictions are categorized as follows:</p>
<ul>
<li><strong>intergenic:</strong> <a href="http://sequenceontology.org/browser/current_release/term/SO:0001628">intergenic_variant</a></li>
<li><strong>upstream/downstream of gene:</strong> <a href="http://sequenceontology.org/browser/current_release/term/SO:0001631">upstream_gene_variant</a>, <a href="http://sequenceontology.org/browser/current_release/term/SO:0001632">downstream_gene_variant</a></li>
<li><strong>5' or 3' UTR:</strong> <a href="http://sequenceontology.org/browser/current_release/term/SO:0001623">5_prime_UTR_variant</a>, <a href="http://sequenceontology.org/browser/current_release/term/SO:0001624">3_prime_UTR_variant</a></li>
<li><strong>CDS - synonymous coding change:</strong> <a href="http://sequenceontology.org/browser/current_release/term/SO:0001819">synonymous_variant</a></li>
<li><strong>CDS - non-synonymous (missense, stop gain/loss, frameshift etc):</strong> <a href="http://sequenceontology.org/browser/current_release/term/SO:0001583">missense_variant</a>, <a href="http://sequenceontology.org/browser/current_release/term/SO:0001821">inframe_insertion</a>, <a href="http://sequenceontology.org/browser/current_release/term/SO:0001822">inframe_deletion</a>, <a href="http://sequenceontology.org/browser/current_release/term/SO:0001589">frameshift_variant</a>, <a href="http://sequenceontology.org/browser/current_release/term/SO:0001582">initiator_codon_variant</a>, <a href="http://sequenceontology.org/browser/current_release/term/SO:0001626">incomplete_terminal_codon_variant</a>, <a href="http://sequenceontology.org/browser/current_release/term/SO:0001578">stop_lost</a>, <a href="http://sequenceontology.org/browser/current_release/term/SO:0001567">stop_retained_variant</a>, <a href="http://sequenceontology.org/browser/current_release/term/SO:0001587">stop_gained</a>,<a href="http://sequenceontology.org/browser/current_release/term/SO:0001621">NMD_transcript_variant</a></li>
<li><strong>intron:</strong> <a href="http://sequenceontology.org/browser/current_release/term/SO:0001627">intron_variant</a></li>
<li><strong>splice site or splice region:</strong> <a href="http://sequenceontology.org/browser/current_release/term/SO:0001575">splice_donor_variant</a>, <a href="http://sequenceontology.org/browser/current_release/term/SO:0001574">splice_acceptor_variant</a>, <a href="http://sequenceontology.org/browser/current_release/term/SO:0001630">splice_region_variant</a></li>
<li><strong>exon of non-coding gene:</strong> <a href="http://sequenceontology.org/browser/current_release/term/SO:0001792">non_coding_exon_variant</a></li>
</ul>
        </div>
        <!-- <div class="tool-box">
          <a href="#" class="circle-icon small glyphicon glyphicon-share-alt"></a>
          <a href="#" class="circle-icon small red-hover glyphicon glyphicon-remove"></a>
          <a href="#" class="circle-icon small red-hover glyphicon glyphicon-flag"></a>
        </div> -->
    </ul>
  </div>
</div>

<script>
  $('.nano').overlayScrollbars({
    scrollbars: {
      autoHide: "leave",
    }
  });
  $('.trigger-message-close').on('click', function () {
    $('body').removeClass('show-message');
    $('#main .message-list li').removeClass('active');
    messageIsOpen = false;
    $('body').removeClass('show-main-overlay');
  });
  $('#message .header .left a').on('click', function (e) {
    e.preventDefault();
    helper.setQuery().clearRefinements().addDisjunctiveFacetRefinement("category", e.target.text).search();
    
  });
  $('#message .header .right a').on('click', function (e) {
    e.preventDefault();
    helper.setQuery().clearRefinements().addFacetRefinement("tags", e.target.text).search();
  });
</script>